tempelis/config/parser_test.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 The Kubernetes Authors.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
```

Comment:
I think the newlines may be important to the formatting. Probably best to leave this alone.

Reply:
agreed